website/docs/user_guide/rbac.md

```diff
@@ -66,7 +66,7 @@ A user group consists of the following:
 
 - a **name** (required)
 - a **description** (optional)
-- one or more users. At least one user must have the owner role
+- a list of users (optional)
```

Comment:
This is intentional, we no longer have any restrictions on the number of users (no users is viable) and the owner concept has been removed in the recent updates